### DISCUSSION TITLE
MegaMenu finite Control

### DISCUSSION THREAD

**Discussion by urbanrogue** [primefaces/community#282] [PrimeVue]
Still kind of new to Primevue and im trying to  customize the look of the mega menu but its not working

Im using PT (passthrough) and I want to pinpoint the submenu (ul) of the 2nd column only at level 1. Basically because I only want items in the 2nd column to use list-disc

Sadly I cant seem to find much documentation on specifics of how to pinpoint a certain item.

Any help would be appreciated.
Here is what I have in PT so far that doesnt work.

 submenu: ({ instance }) => ({
                class: [
                  'm-0 w-full', 'py-1 w-48',
                 { 'list-disc': instance.level == 1 && instance.columnIndex == 2 }
                ]
            }),
            submenu

**Comment by tugcekucukoglu** [comment]
columnIndex is not a valid option. Available options list here: https://primevue.org/megamenu/#api.megamenu.interfaces.MegaMenuContext

**Reply by urbanrogue** [reply]
Thanks.  I found that I had to fork the megamenu files to my own component so I can customize it further. As much as I dislike having to do that.  As some columns needed to display discs,  some needed red left borders. some text beside other text needed to be bold. etc...
